RuneApp/Main.cs

```diff
@@ -1033,19 +1038,22 @@ public partial class Main : Form {
 
 		private void tsBtnBuildsResume_Click(object sender, EventArgs e) {
 			Program.RunBuilds(true, -1);
-			tsBtnBuildsRunOne.DropDown.Close();
+			tsBtnBuildsRun.DropDown.Close();
 		}
 
 		private void tsBtnBuildsRunUpTo_Click(object sender, EventArgs e) {
-			int selected = -1;
 			if (buildList.SelectedItems.Count > 0)
-				selected = (buildList.SelectedItems[0].Tag as Build).Priority;
-			Program.RunBuilds(true, selected);
+            {
```

Comment:
What is your IDE/git config for whitespace?

Reply:
Or maybe this was during my Tab phase and you are using 4 spaces 👀

Reply:
I'm using whatever VS Community defaults to.